boards/xtensa/waveshare_esp32s3_touch_lcd/waveshare_esp32s3_touch_lcd.dts

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright (c) 2024 Joel Guittet
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <espressif/esp32s3/esp32s3_r2.dtsi>
+#include "waveshare_esp32s3_touch_lcd-pinctrl.dtsi"
+#include <zephyr/dt-bindings/display/gc9x01x.h>
```

Comment:
missed a dep?

Reply:
Oh, https://github.com/zephyrproject-rtos/zephyr/pull/67012, I see that now.

Reply:
Yes exactly, this is why some twistter tests are failing here